packages/react-reconciler/src/ReactFiberContext.js

```diff
@@ -49,11 +49,11 @@ let previousContext: Object = emptyContextObject;
 function getUnmaskedContext(
   workInProgress: Fiber,
   Component: Function,
+  didPushOwnContext: boolean,
```

Comment:
nit: The naming of this implies that the Fiber _is_ a context provider when it's actually meant more to reflect which phase the component is in. Maybe we could call it something more meaningful? Even `didPushOwnContextIfProvider`?

Reply:
Sounds good. I couldn't find a name I'd be happy with. Which reflects how messed up this whole thing is.